docs/docusaurus/docs/reference/learn/data_quality_use_cases/integrity.md

```diff
@@ -0,0 +1,278 @@
+
+Consider these common data integrity challenges:
+
+* **Referential integrity**: Ensuring that relationships between tables remain valid and consistent is crucial. For example, in a financial system, every transaction should link to valid accounts, and account balances should accurately reflect all related transactions.
+* **Business rule compliance**: Data must adhere to established business logic and domain-specific constraints. An e-commerce platform, for instance, should validate that ordered quantities don't exceed available inventory and that shipping addresses match customer records.
+* **Cross-column consistency**: Logical relationships between related columns within and across tables must be verified. In a healthcare system, diagnosis codes should align with prescribed medications, and treatment dates should fall within the authorized timeframe.
+* **Value dependencies**: Hierarchies and dependencies in numerical and categorical data must be maintained. For example, a project management system should ensure that task start dates precede end dates and that parent tasks span the duration of their subtasks.
+
+Failing to validate these relationships can result in data discrepancies, incorrect calculations, and flawed insights, undermining the reliability and value of your data assets.
+
+Great Expectations (GX) offers a robust framework for validating data integrity at scale. Its expressive Expectations, flexible validation workflows, and support for diverse data sources enable data teams to implement comprehensive integrity checks across their entire data ecosystem.
+
+In this guide, we'll demonstrate how to use GX to define and enforce data integrity rules through practical examples and best practices. We'll cover techniques for validating referential integrity, cross-table consistency, and compliance with business rules.
+
+## Prerequisite knowledge
+
+This article assumes basic familiarity with GX components and workflows. If you're new to GX, start with the [GX Cloud](https://docs.greatexpectations.io/docs/cloud/overview/gx_cloud_overview) and [GX Core](https://docs.greatexpectations.io/docs/guides/overview) overviews to familiarize yourself with key concepts and setup procedures.
+
+## Data preview
+
+The examples in this guide use a sample dataset of financial transactions and transfer balances, provided from public Postgres database tables. The sample data is also available in CSV format for each table:
+
+
+1. `transactions` table ([CSV](https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/learn_data_quality_use_cases/integrity_transactions.csv)):
+
+```
+| transaction_id | sender_id | recipient_id | amount  | transaction_date | reference_number | confirmation_code |
```

Comment:
What's the different between `reference_number` and `confirmation_code`?

Reply:
while these values have different semantic meanings, they happen to be numerically equivalent in this example. i'm using this overlap to demonstrate integrity constraints in a simplified single-table scenario.